apps/meteor/app/livechat/server/api/v1/config.ts

```diff
@@ -12,7 +13,7 @@ API.v1.addRoute(
 	{ validateParams: isGETLivechatConfigParams },
 	{
 		async get() {
-			const enabled = Livechat.enabled();
+			const enabled = serverSettings.get('Livechat_enabled');
```

Comment:
```suggestion
			const enabled = serverSettings.get<boolean>('Livechat_enabled');
```
Not a big deal, but it'd be good to have correct typing here as we had before